.github/workflows/TODO_TO_ISSUE.yml

```
@@ -0,0 +1,36 @@
+      - name: 📝 Create TODOs
+        uses: alstr/todo-to-issue-action@v5
+        with:
+          INSERT_ISSUE_URLS: 'true'
+
+      - name: Set Git user
+        run: |
+          git config --global user.name "github-actions[bot]"
+          git config --global user.email "github-actions[bot]@users.noreply.github.com"
+
+      - name: Commit and Push Changes
+        run: |
```

actionlint

shellcheck reported issue in this script: SC2006:style:2:7: Use $(...) notation instead of legacy backticks `...`

(shellcheck)